openedx/core/djangoapps/user_api/views.py

```diff
@@ -479,7 +487,34 @@ def _add_password_field(self, form_desc, required=True):
                 "min_length": PASSWORD_MIN_LENGTH,
                 "max_length": PASSWORD_MAX_LENGTH,
             },
-            required=required
+            required=required,
+            placeholder='Password'
+        )
+
+    def _add_confirm_password_field(self, form_desc, required=True):
+        """Add a confirm password field to a form description.
+
+        Arguments:
+            form_desc: A form description
+
+        Keyword Arguments:
+            required (bool): Whether this field is required; defaults to True
+
+        """
+        # Translators: This label appears above a field on the registration form
+        # meant to hold the user's password.
+        password_label = _(u"Confirm Password")
+
+        form_desc.add_field(
+            "confirm_password",
+            label=password_label,
```

Comment:
no validation if confirm password is same as passowrod?

Reply:
Added in front-end